bitnami/scylladb/templates/hpa.yaml

```diff
@@ -34,15 +34,13 @@ spec:
         name: cpu
         target:
           averageUtilization: {{ .Values.autoscaling.hpa.targetCPUUtilizationPercentage }}
-          type: Utilization
```

Comment:
Is this intentional? This was present in other charts like nginx and hasn't been removed.